runtime/lua/tohtml.lua

```diff
@@ -1293,6 +1293,9 @@ local function opt_to_global_state(opt, title)
   elseif vim.o.guifont:match('^[^:]+') then
     table.insert(fonts, vim.o.guifont:match('^[^:]+'))
   end
+  for i, v in pairs(fonts) do
+    fonts[i] = ('"%s"'):format(v)
+  end
```

Comment:
since we're already doing a table.concat below, perhaps it makes sense to do this there. instead of storing this "formatting" info in the `fonts` list (although it's short lived so doesn't technically matter currently)

    font = ('"%s"'):format(table.concat(fonts, '","'))

Reply:
Generic family names (`monospace` here) must not be quoted because the browser recognizes them as font families. so, I think we should enclose specific font families in quotation marks before adding `monospace` to the `fonts`.

Reply:
then we need to call that out in a code comment

Reply:
When multiple fonts are specified in `vim.o.guifont`, they are currently being inserted as a single element. It seems we need to modify this so that each font is inserted as a separate element.

and, Is it appropriate to write the comment like this above the for loop?
```
-- Generic family names (monospace here) must not be quoted
-- because the browser recognizes them as font families.
```

Reply:
I mistakenly thought it was about specifying multiple fonts in `vim.o.guifont`.(A fix is necessary for the above problem.)
If `monospace` is specified in `vim.o.guifont`, it is treated as a regular font family name, resulting in something like:
```
{font-family: "monospace",monospace}
```
It is unlikely that a generic family will be specified in `vim.o.guifont`.